examples/js/archive.js

```diff
@@ -281,87 +280,121 @@ function isJsonDetected(url) {
 }
 
 // places image on tile
-function placeImage (imageURL, options, newImage = false) {
+function placeImage(imageURL, options, newImage = false) {
   let image;
-  
-  if (newImage) { // construct new map
-    image = L.distortableImageOverlay(
-      imageURL,
-      {tooltipText: options.tooltipText}
-    );
-  } else { // reconstruct map to previous saved state
+
+  if (newImage) {
+    // construct new map
+    image = L.distortableImageOverlay(imageURL, {
+      tooltipText: options.tooltipText,
+    });
+  } else {
+    // reconstruct map to previous saved state
     const zc = options.corners;
-    const reOrderedCorners = [ // coordinates passed in correct order NW, NE, SW, SE (i.e., 'Z' pattern)
-      {lat: zc[0].lat, lon: zc[0].lon || zc[0].lng},
-      {lat: zc[1].lat, lon: zc[1].lon || zc[1].lng},
-      {lat: zc[3].lat, lon: zc[3].lon || zc[3].lng},
-      {lat: zc[2].lat, lon: zc[2].lon || zc[2].lng}
+    const reOrderedCorners = [
+      // coordinates passed in correct order NW, NE, SW, SE (i.e., 'Z' pattern)
+      { lat: zc[0].lat, lon: zc[0].lon || zc[0].lng },
+      { lat: zc[1].lat, lon: zc[1].lon || zc[1].lng },
+      { lat: zc[3].lat, lon: zc[3].lon || zc[3].lng },
+      { lat: zc[2].lat, lon: zc[2].lon || zc[2].lng },
     ];
 
-    image = L.distortableImageOverlay(
-      imageURL,
-      {
-        tooltipText: options.tooltipText,
-        corners: reOrderedCorners, 
-      }
-    );
+    image = L.distortableImageOverlay(imageURL, {
+      tooltipText: options.tooltipText,
+      corners: reOrderedCorners,
+    });
   }
 
   map.imgGroup.addLayer(image);
 }
 
 // Reconstruct Map from JSON URL (image objects without coordinates are ignored)
-const reconstructMapFromJson = async (jsonDownloadURL) => {
+const reconstructMapFromJson = async (jsonDownloadURL = false, savedMapObj) => {
```

Comment:
@jywarren Here, all I did was make `reconstructMapFromJson()`  reusable, for some reason by making `jsonDownloadURL` a default parameter

I wonder why it shows other code i didn't add 🤔

Reply:
it looks like it's showing formatting changes from an auto-formatter. do you have one set up?

Reply:
Oh that's true, I must've forgotten to turn it off
I'll see how to revert it